canopen/emcy.py

```diff
@@ -52,7 +53,7 @@ def reset(self):
 
     def wait(
         self, emcy_code: Optional[int] = None, timeout: float = 10
-    ) -> "EmcyError":
+    ) -> Optional[EmcyError]:
```

Comment:
Cosmetics, let's keep these separate from the mypy error fixing.  There are lots of things like this to do, but better to keep a PR focused and revisit such cleanups later. IMHO.